modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/dhs.scala

```diff
@@ -0,0 +1,8 @@
+package edu.gemini.seqexec.model
+
+object dhs {
+
+  // Common data types
+  type ObsId = String
```

Comment:
Yes, it is a placeholder. It was on the original seqexec server and I moved it here to make it shared between the web server and seqexec server
It should be a `SPObservationID` but that's a Java class